desktop/apps/editorial_features/components/gucci/templates/index.jade

```diff
@@ -7,6 +7,13 @@ block head
   include head
 
 block body
+  a(href="https://ad.doubleclick.net/ddm/jump/N32001.3019648ARTSY/B20483079.208849246;sz=1x1;ord#{moment().unix()}=?")
+    img(
+      src="https://ad.doubleclick.net/ddm/ad/N32001.3019648ARTSY/B20483079.208849246;sz=1x1;ord=#{moment().unix()};dc_lat=;dc_rdid=;tag_for_child_directed_treatment=?"
+      width='1'
+      height='1'
+    )
+
```

Comment:
Moved this out of the main React body since some re-renders were causing the pixel to be loaded multiple times.

Reply:
Makes sense to me!